src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -222,12 +172,64 @@ public async Task<T> GetConfigurationAsync(CancellationToken cancel)
             }
         }
 
+        /// <summary>
+        /// Obtains an updated version of Configuration.
+        /// </summary>
+        /// <param name="cancel">CancellationToken</param>
+        /// <returns>Configuration of type StandardConfiguration.</returns>
+        /// <remarks>If the time since the last call is less than <see cref="StandardConfigurationManager.AutomaticRefreshInterval"/> then <see cref="IConfigurationRetriever{T}.GetConfigurationAsync"/> is not called and the current Configuration is returned.</remarks>
+        public override async Task<StandardConfiguration> GetGeneralConfigurationAsync(CancellationToken cancel)
```

Comment:
Unfortunately the name of this method cannot be the same as GetConfigurationAsync(...), as having a different return type does not count as an overload.